lesson20/src/js/index.js

```diff
@@ -0,0 +1,93 @@
+import { createAttributedElements } from './utiles/createAttributeWithEllement';
```

Comment:
nits
```suggestion
import { createAttributedElements } from './utils/createAttributeWithElement';
```

Reply:
Thank you for your review!
Corrected the spelling mistake.

[7bd6683](https://github.com/nyattsu72/hanson-frontend-cource/pull/28/commits/7bd6683222e5ad91d34a1312350245d60c494ae0)